Scripts/BuildPhases/runRubyScript

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+
+# Use this to run a Ruby script from a "Script Build Phase" from Xcode.
+#
+# Since shell scripts ran by Xcode  do not source the user shell profile, typical user setups like the configurations of `rbenv` or `rvm` would not be set up properly.
+# This script check if either rbenv or rvm is installed on the Mac and runs the setup steps as appririate, before running the ruby script via `bundle exec`
+#
+# Usage:
+#   `runRubyScript <script_name.rb> <optional_args>`
+#
+# Where <script_name.rb` can be either an absolute path, or a path relative to this runRubyScript wrapper script.
+#
+# Inspiration: https://mgrebenets.github.io/xcode/2019/04/04/xcode-build-phases-and-environment
+#
+
+# Add `rbenv` and `rvm` binaries to PATH, so that we support both
+export PATH="$HOME/.rbenv/shims:$HOME/.rvm/bin:$PATH"
```

Comment:
At first I tried to add some logic to _detect_ which of `rbenv` or `rvm` were installed, and call the setup commands appropriately for each and all (see attempt in parent commit a84b805e1a37d23c14199cc1cb903fb4bd4b6192) but that didn't work. E.g. `command -v rbenv` apparently failed to detect that `rbenv` was installed, even system-wise, when ran by Xcode — while it worked in Terminal — so it didn't go into the `if` in Tanner's Mac… (even though on my own Mac it did work)

So in the end I settled for a simpler solution after all, which is to add the paths to the shims and binaries for both cases in `$PATH` here to cover both bases, and then calling `bundle exec ruby <script.rb>` — which would, in both cases (`rvm` like `rbenv`), use the shims and thus the locally installed ruby instead of the system one (while I _think_ that just `bundle exec <script.rb>` would have used the shebang at the to of the `script.rb`, and in such case I'm not sure which one `/usr/bin/env ruby` would end up using…? 🤷 )